host_core/test/e2e/echo_test.exs

```diff
@@ -5,7 +5,9 @@ defmodule HostCore.E2E.EchoTest do
   setup do
     {:ok, evt_watcher} =
       GenServer.start_link(HostCoreTest.EventWatcher, HostCore.Host.lattice_prefix())
-
+    on_exit(fn ->
+      :ets.delete_all_objects(:linkdef_table)
+    end)
```

Comment:
The EventWatcher will actually take care of this on initialization of each test, it's a fine safety measure but not necessary